{"status": "broken", "name": "Excluding connected nodes", "labels": [{"name": "severity", "value": "normal"}, {"name": "tag", "value": "skip"}, {"name": "feature", "value": "MatchAcceptance2"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython2"}], "stop": 1578486064158, "historyId": "026362814f46504390a6be6550a10a8f", "start": 1578486063951, "steps": [{"status": "passed", "start": 1578486063951, "stop": 1578486064156, "name": "Given an empty graph"}, {"status": "passed", "start": 1578486064156, "stop": 1578486064157, "name": "And having executed", "attachments": [{"source": "ce523d09-e0cb-4a88-9a35-45cd1e6721bc-attachment.txt", "type": "text/plain", "name": ".text"}]}, {"status": "passed", "start": 1578486064157, "stop": 1578486064157, "name": "When executing query", "attachments": [{"source": "8d9cd271-9475-4b48-81eb-d546573e4987-attachment.txt", "type": "text/plain", "name": ".text"}]}, {"status": "broken", "attachments": [{"source": "a2caebcd-79c6-4473-95d3-4b6378f13f3c-attachment.csv", "type": "text/csv", "name": ".table"}], "stop": 1578486064158, "start": 1578486064157, "statusDetails": {"message": "ResponseError: RedisGraph does not support OPTIONAL MATCH.\n", "trace": "  File \"/home/laptop-linux/.local/lib/python2.7/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/home/laptop-linux/.local/lib/python2.7/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features/steps/Steps.py\", line 84, in step_impl\n    raise exception\n"}, "name": "Then the result should be"}, {"status": "skipped", "start": 1578486064158, "stop": 1578486064158, "name": "And no side effects"}], "statusDetails": {"message": "ResponseError: RedisGraph does not support OPTIONAL MATCH.\n", "trace": "  File \"/home/laptop-linux/.local/lib/python2.7/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/home/laptop-linux/.local/lib/python2.7/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features/steps/Steps.py\", line 84, in step_impl\n    raise exception\n"}, "uuid": "dfe55e42-d7e7-45d1-b69b-ce50fcdaeea7"}